community/apt-dater-host/apk-improve-Machine-Type-reporting.patch

```diff
@@ -0,0 +1,62 @@
+From f90d99ca8c4a04396d94ef42abe794d6ce75173b Mon Sep 17 00:00:00 2001
```

Comment:
Where is this patch from? Is it backported from upstream? Please clarify it in the patch.

Reply:
yes already merged upstream.

Reply:
https://github.com/DE-IBH/apt-dater-host/pull/24